pkg/types/validation/installconfig_test.go

```diff
@@ -590,7 +590,7 @@ func TestValidateInstallConfig(t *testing.T) {
 				c.Platform = types.Platform{}
 				return c
 			}(),
-			expectedError: `^platform: Invalid value: "": must specify one of the platforms \(alibabacloud, aws, azure, baremetal, gcp, ibmcloud, none, nutanix, openstack, ovirt, powervs, vsphere\)$`,
+			expectedError: `^platform: Invalid value: "": must specify one of the platforms \(alibabacloud, aws, azure, baremetal, gcp, ibmcloud, none, nutanix, openstack, powervs, vsphere\)$`,
```

Comment:
Have these tests been validated? I don't see where the error message has been changed to reflect this.

Reply:
What do you mean by validated? The unit tests are being run by several steps in the pipeline. It is testing `ValidateInstallConfig`. I also calls `validatePlatform`, which throws that error [here](https://github.com/openshift/installer/blob/master/pkg/types/validation/installconfig.go#L713) and, in turn, uses a [slice of PlatformNames](https://github.com/openshift/installer/blob/master/pkg/types/installconfig.go#L37-L48) to build it.